internal/pkg/vm/actor/builtin/miner/miner.go

```diff
@@ -730,7 +730,7 @@ func (a *Impl) CommitSector(ctx invocationContext, sectorID uint64, commD, commR
 			copy(req.CommR[:], commR)
 			copy(req.CommRStar[:], commRStar)
 			req.Proof = proof
-			req.ProverID = sectorbuilder.AddressToProverID(ctx.Message().To)
+			req.ProverID = sectorbuilder.AddressToProverID(ctx.OriginalMessage().To)
```

Comment:
FYI I'm adding `Receiver` to the spec, which can replace this use of message

Reply:
👍 thanks!